File: Level3/Materal.BusinessFlow/Src/Materal.BusinessFlow.Abstractions/Services/Models/Node/QueryNodeModel.cs
using Materal.Utils.Model;

namespace Materal.BusinessFlow.Abstractions.Services.Models.Node
{
    public class QueryNodeModel : BaseQueryModel
    {
        /// <summary>
        /// 名称
        /// </summary>
        [Contains]
        public string? Name { get; set; }
        /// <summary>
        /// 处理类型
        /// </summary>
        [Equal]
        public NodeHandleTypeEnum? HandleType { get; set; }
        /// <summary>
        /// 步骤唯一标识
        /// </summary>
        [Equal]
        public Guid? StepID { get; set; }
    }
}

File: Level3/Materal.BusinessFlow/Demo/BusinessFlowWebDemo/src/models/Node/QueryNodeModel.ts
import { PageRequestModel } from "../PageRequestModel";
import { NodeHandleTypeEnum } from "./NodeHandleTypeEnum";

export class QueryNodeModel extends PageRequestModel {
    /**
     * 名称
     */
    public Name?: string;
    /**
     * 步骤唯一标识
     */
    public StepID?: string;
    /**
     * 处理类型
     */
    public HandleType?: NodeHandleTypeEnum;
}